DEPENDENCIES.md

```diff
@@ -1,6 +1,6 @@
 # Licenses for Third-Party Dependencies
 
-Software packages built from this source code may incorporate code from a number of third-party dependencies.
+Binary distributions of this software incorporate code from a number of third-party dependencies.
```

Comment:
Reworded so it reads a little better when this file accompanies a binary distribution.